app/distance.py

```diff
@@ -0,0 +1,11 @@
+class Distance:
+    def __init__(self, start: list, end: list):
+        self._start = start
+        self._end = end
+
+    def distance(self):
+        first_addition = (self._end[0] - self._start[0]) ** 2

```

Comment:
it s  correct, and i started do this the same, but in method dist is in math module, you could use this